.buildkite/pipeline.yml

```diff
@@ -241,6 +239,12 @@ steps:
       JULIA_CUDA_USE_BINARYBUILDER: 'true'
     if: build.message !~ /\[skip tests\]/ && !build.pull_request.draft
     timeout_in_minutes: 120
+    commands: |
+      export JULIA_PROJECT="CUTENSORNET"
```

Comment:
Yeah, `JULIA_PROJECT` and `--project` are basically equivalent.

But, I just made a new suggestion that doesn't use `JULIA_PROJECT` or `--project`. Because we need to `dev` both `CUDA` and `CUTENSORNET` (resp. `CUSTATEVEC`) in the same environment.

Reply:
Yeah. We could try to add a feature to the Buildkite plugin that would automatically `Pkg.dev` the dependencies, but this is a relatively uncommon use case, so I don't know if it's worth doing.